src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,6 +127,11 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void sortPersonList() {
+        addressBook.sort();
```

Comment:
You might not want to mess with the addressBook directly. Try looking into modifying the `filteredPersons` variable instead, as it will improve the defensiveness of our code as well.